tox.ini

```diff
@@ -1,28 +1,17 @@
 [tox]
-envlist = py26,py27,pypy,py26-trunkdeps,py27-trunkdeps,pypy-trunkdeps
+envlist = py26-{twlatest,tw150},{py26,py27,pypy}-{twtrunk,twlatest,tw150}
```

Comment:
break the 2.6 build? perhaps we should just delete 2.6 support when that happens?